packages/dev/core/src/Materials/greasedLinePluginMaterial.ts

```diff
@@ -64,7 +96,7 @@ export interface GreasedLineMaterialOptions {
      * Colors of the line segments.
      * Defaults to empty.
      */
-    colors?: Color3[];
+    colors?: Nullable<Color3[]>;

```

Comment:
nit - no need for both optional and Nullable. one of the two is enough.

Reply:
`export type Nullable<T> = T | null;`

The `colors` option is optional so the `?` is required here and it can have a value of `null` as well to remove the color table texture.

Reply:
right. assigning `null` right here will remove the need for "undefined".
```suggestion
    colors: Nullable<Color3[]> = null;
```

Reply:
I think you didn't realize that this is line is inside an interface definition, so there have to be a question mark to make the interface member optional.

Reply:
you are right, i didn't notice it is in an interface :-) It is, however, not optional. It is nullable :-)

I don't see the reason to make is both nullable and optional. however, i am not going to die on this hill. If you can't find a way to remove the optional, keep it for now.

Reply:
I started to code in TS only 3 years ago so maybe I am missing something here. :-)
The question mark makes it optional so I can pass an object without this interface member, right?
The `Nullable` type allows to set a `null` value to it.

Or is it enough just to mark it in the interface optional with the question mark to make it receiving `null` value as well? Never tried it before :-D

Reply:
Let's have an example:
```
interface MyInterface {
  color?: Color3
}

class MyClass {
  constuctor(public options: MyInterface) { 
  }
}
```

Now I can create the class it without the `color` property:
`const clazz = new MyClass({})`

Now I want to set the value of the `color` to `null`:
```
clazz.options.color = null // fails, accepts only undefined
```

Right?

So I have to mark it `Nullable` as well.

I could copy the `options` object to another object in the constructor. This object could have a `color` property only `Nullable` (no need to be optional) and I could set it automatically to `null` if the object passed to the constructor had no `color` property specified.

Reply:
@sebavan @RaananW 

Guys!

I've modified the code amd removed the `undefined` stuff :-D 
I'll push it to the repo after I test all the PGs from the docs page.